file2.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE users(
+	username       TEXT PRIMARY KEY NOT NULL,
```

Comment:
I think using username as a primary key could cause really big problems for us. Using an ID would be more performant, and using a natural key like username would be very non-standard. Also, since usernames can change, you would have to change all the foreign keyed tables, which would also be extremely costly performance wise.